### PR TITLE
Use Java 11 to build and develop the plugin

### PR DESCRIPTION
**Pull request by Undin**
It's required to build the plugin for 2020.3 platform

### CODE REVIEW

**Comment by Undin** [comment]
bors retry

**Comment by Undin** [comment]
bors retry

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -14,6 +14,12 @@ IDE.
 
 # Getting started
 
+## Environment
+
+Java 11 is required for development.
+For example, you can install [openJDK](https://openjdk.java.net/install/) or [Amazon Corretto](https://docs.aws.amazon.com/corretto/latest/corretto-11-ug/downloads-list.html).
+You also [install](https://www.jetbrains.com/help/idea/sdk.html#set-up-jdk) Java just from your IntelliJ IDEA.
```
Comment (vlad20012) [inline CONTRIBUTING.md:21]: You `can` also?

Comment (Undin) [reply CONTRIBUTING.md:21]: Yep
Thanks!

**Pull request merged**